platform/jvm/capture/src/test/kotlin/io/bitdrift/capture/CaptureLoggerTest.kt

```diff
@@ -125,7 +125,6 @@ class CaptureLoggerTest {
             "_host" to "api.bitdrift.io",
             "_method" to "GET",
             "_path" to "/my_path/12345",
-            "_path_template" to "/my_path/<id>",
```

Comment:
can we now remove the `CaptureJniLibrary.load()` and maybe even make it a plain unit test (vs robolectric one)?